requirements.txt

```diff
@@ -1,10 +1,10 @@
-loguru~=0.5.2
-PyYAML~=5.3.1
-fuzzywuzzy~=0.18.0
-python-Levenshtein~=0.12.0
+loguru
+PyYAML
+fuzzywuzzy
+python-Levenshtein
 pyelftools~=0.26
-enlighten~=1.6.2
-tqdm~=4.50.2
+enlighten
+tqdm
 graphlib-backport
-networkx~=2.5
-jsonschema~=3.2.0
+networkx
+jsonschema
```

Comment:
This is usually a footgun.
You sure?

Reply:
I did this because `~=V.N` is ~equivalent to `>= V.N, == V.*` and orchestra is a globally installed package, so if there are packages requiring an incompatible minor version of some orchestra dependency a conflict will be generated (anecdotally, this happened on my system). However, there is indeed the risk of API-breaking updates, so I just limited myself to bumping the version number.

Reply:
Bumping and testing >>> having arbitrary future deps